lib/track.js

```diff
@@ -144,6 +144,16 @@ Track.prototype.load = function() {
   return this._send('track', 'PAGE_LOADED');
 };
 
+/**
+ * event sends a user-driven event to Ravelin.
+ * @param {string} action The action that was performed.
+ * @param {object} [props] Additional properties to describe the event.
+ * @returns
+ */
+Track.prototype.event = function(action, props) {
+  return this._send('track', action, props);
```

Comment:
Just a suggestion, to keep it consistent.

```suggestion
 * @param {string} name The name of the event.
 * @param {object} [props] Additional properties to describe the event.
 * @returns
 */
Track.prototype.event = function(name, props) {
  return this._send('track', name, props);
```